rocks-bin/src/check.rs

```diff
@@ -0,0 +1,42 @@
+use eyre::{OptionExt, Result};
+use rocks_lib::{
+    build::BuildBehaviour,
+    config::{Config, LuaVersion},
+    lockfile::PinnedState::Pinned,
+    operations::{self, install},
+    progress::MultiProgress,
+    project::Project,
+    remote_package_db::RemotePackageDB,
+};
+
+pub async fn check(config: Config) -> Result<()> {
+    let project = Project::current()?.ok_or_eyre("Not in a project!")?;
+
+    let db = RemotePackageDB::from_config(&config).await?;
+
+    install(
+        vec![(BuildBehaviour::NoForce, "luacheck".parse()?)],
+        Pinned,
+        &db,
+        &config,
+        MultiProgress::new_arc(),
+    )
+    .await?;
+
+    operations::run(
+        "luacheck",
+        vec![
+            project.root().to_string_lossy().into(),
+            "--exclude-files".into(),
+            project
+                .tree(LuaVersion::from(&config)?)?
+                .root()
+                .to_string_lossy()
+                .to_string(),
+        ],
```

Comment:
suggestion:

```suggestion
        vec![
            format!("{}", project.root().display()),
            "--exclude-files".into(),
            format!("{}", project
                .tree(LuaVersion::from(&config)?)?
                .root()
                .display()),
        ],
```

Reply:
Not sure if that's actually better, but apparently `display()` is safer.

Reply:
`display()` does essentially the same thing as `to_string_lossy` by replacing all invalid unicode chars with the replacement character. Amounts to the same code in the end :)